Dockerfile

```diff
@@ -120,6 +120,8 @@ RUN rm -rf ${KPI_NODE_PATH} && \
     rm -rf "${KPI_SRC_DIR}/jsapp/fonts" && \
     rm -rf "${KPI_SRC_DIR}/jsapp/compiled" && \
     npm install --quiet && \
+    mkdir -p "${TMP_DIR}/.npm" && \
+    npm config set cache "${TMP_DIR}/.npm" --global && \
```

Comment:
Looks fine!

- Slight preference to see these lines placed upwards, before `npm install -g npm@8.5.5`, if the fix still works that way. My reasoning is that I think `npm install -g npm@8.5.5` will still place stuff into the default cache folder, and then if `npm config set cache` doesn't clean it up, those files would hang around in the Docker image.